src/System.Collections/tests/Generic/Dictionary/Dictionary.Generic.Tests.netcoreapp.cs

```diff
@@ -91,67 +92,107 @@ public void Dictionary_Generic_RemoveKey_DefaultKeyContainedInDictionary(int cou
         public void EnsureCapacity_NegativeCapacityRequested_Throws()
         {
             var dictionary = new Dictionary<string, string>();
-            Assert.Throws<ArgumentOutOfRangeException>(() => dictionary.EnsureCapacity(-1));
+            AssertExtensions.Throws<ArgumentOutOfRangeException>("capacity", () => dictionary.EnsureCapacity(-1));
```

Comment:
Improving Assert.Throws to AssertExtensions so we can assert ex.ParamName as well.